README.md

```diff
@@ -1,5 +1,5 @@
 # Open Energy Dashboard #
-[![Build Status](https://travis-ci.org/beloitcollegecomputerscience/OED.svg?branch=master)](https://travis-ci.org/beloitcollegecomputerscience/OED)
+[![Build Status](https://travis-ci.org/OpenEnergyDashboard/OED.svg?branch=master)](https://travis-ci.org/OpenEnergyDashboard/OED)
```

Comment:
The new build status logo says "build: Unknown"

Reply:
Yes, because we're not currently building OpenEnergyDashboard/OED on Travis...